fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalScanOperator.java

```diff
@@ -93,6 +95,14 @@ public String toString() {
                 '}';
     }
 
+    public ScanOperatorPredicates getScanOperatorPredicates() throws AnalysisException {
+        throw new AnalysisException("Base class calls are not supported");
```

Comment:
Base class calls are not supported -> Operation getScanOperatorPredicates are not supported for this ScanOperator!

Reply:
done